flaskr.py

```diff
@@ -1,30 +1,31 @@
 import os
 import sqlite3
+import fnmatch
 from flask import Flask, request, session, g, redirect, url_for, abort, render_template, flash
 
 app = Flask(__name__)
 app.config.from_object(__name__)
 
 # Load default config and override config from an environment varible
 app.config.update(dict(
-	DATABASE = os.path.join(app.root_path, 'flaskr.db'),
-	SECRET_KEY = 'development key',
-	USERNAME = 'admin',
-	PASSWORD = 'default'
+    DATABASE = os.path.join(app.root_path, 'flaskr.db'),
```

Comment:
Yes, I know right. That's because of the original file had tab-style indentation and I converted it to spaces in order to write space-indented code.